torch/ao/quantization/fx/_model_report/model_report_visualizer.py

```diff
@@ -68,7 +69,8 @@ def get_all_unique_module_fqns(self) -> Set[str]:
         Returns all the unique module fqns present in the reports the ModelReportVisualizer
         instance was initialized with.
         """
-        pass
+        # returns the keys of the ordered dict
+        return set(self.generated_reports.keys())
 
     def get_all_unique_feature_names(self, plottable: bool) -> Set[str]:
```

Comment:
nit: `plottable_features_only`? `skip_non_tensor_values`? `plottable` sounds a bit confusing. Actually why not just default this to true; I don't imagine the user will use this outside of visualization given this is part of `ModelReportVisualizer`